auth/api/iam/types.go

```diff
@@ -99,6 +96,58 @@ var vpFormatsSupported = map[string]map[string][]string{
 // https://openid.bitbucket.io/connect/openid-4-verifiable-presentations-1_0.html#name-verifier-metadata-managemen
 var clientIdSchemesSupported = []string{"did"}
 
+// clientIDParam is the name of the client_id parameter.
+// Specified by https://datatracker.ietf.org/doc/html/rfc6749#section-2.2
+const clientIDParam = "client_id"
+
+// responseTypeParam is the name of the response_type parameter.
+// Specified by https://datatracker.ietf.org/doc/html/rfc6749#section-3.1.1
+const responseTypeParam = "response_type"
+
+// scopeParam is the name of the scope parameter.
+// Specified by https://datatracker.ietf.org/doc/html/rfc6749#section-3.3
+const scopeParam = "scope"
+
+// stateParam is the name of the state parameter.
+// Specified by https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.1
+const stateParam = "state"
+
+// redirectURIParam is the name of the redirect_uri parameter.
+// Specified by https://datatracker.ietf.org/doc/html/rfc6749#section-3.1.2
+const redirectURIParam = "redirect_uri"
+
+// presentationDefParam is the name of the OpenID4VP presentation_definition parameter.
+// Specified by https://openid.bitbucket.io/connect/openid-4-verifiable-presentations-1_0.html#name-presentation_definition-par
+const presentationDefParam = "presentation_definition"
+
+// presentationDefUriParam is the name of the OpenID4VP presentation_definition_uri parameter.
+// Specified by https://openid.bitbucket.io/connect/openid-4-verifiable-presentations-1_0.html#name-presentation_definition_uri
+const presentationDefUriParam = "presentation_definition_uri"
+
+// presentationSubmissionParam is the name of the OpenID4VP presentation_submission parameter.
+// Specified by https://openid.bitbucket.io/connect/openid-4-verifiable-presentations-1_0.html#name-response-parameters
+const presentationSubmissionParam = "presentation_submission"
+
+// clientMetadataParam is the name of the OpenID4VP client_metadata parameter.
+// Specified by https://openid.bitbucket.io/connect/openid-4-verifiable-presentations-1_0.html#name-authorization-request
+const clientMetadataParam = "client_metadata"
+
+// clientMetadataParam is the name of the OpenID4VP client_metadata_uri parameter.
+// Specified by https://openid.bitbucket.io/connect/openid-4-verifiable-presentations-1_0.html#name-authorization-request
+const clientMetadataURIParam = "client_metadata_uri"
+
+// clientIDSchemeParam is the name of the OpenID4VP client_id_scheme parameter.
+// Specified by https://openid.bitbucket.io/connect/openid-4-verifiable-presentations-1_0.html#name-authorization-request
+const clientIDSchemeParam = "client_id_scheme"
+
+// vpTokenParam is the name of the OpenID4VP vp_token parameter.
+// Specified by https://openid.bitbucket.io/connect/openid-4-verifiable-presentations-1_0.html#name-response-type-vp_token
+const vpTokenParam = "vp_token"
```

Comment:
also declared as `responseTypeVPToken` and `grantTypeVPToken`. It is correct, but feels silly to have 3 constants for the same value